cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -137,3 +156,40 @@ private def artifactCleanup(int months, String[] repos, log, paceTimeMS, dryRun
         log.info "Finished cleanup, deleted $foundArtifacts artifacts that took up $bytesFound bytes"
     }
 }
+
+private def getSkippedPaths(String[] repos) {
+    def timeStart = new Date()
+    def skip = [:]
+    for (String repoKey : repos){
+        def pathsTmp = []
+        def aql = "items.find({\"repo\":\"" + repoKey + "\",\"type\": \"any\",\"@cleanup.skip\":\"true\"}).include(\"path\", \"name\")"
+        searches.aql(aql.toString()) {
+            for (item in it) {
```

Comment:
I would append a `/` to folder paths before adding them to the list. Otherwise you could end up with paths like `org/foo` and `org/foobar`, and the latter would be pruned from the list because it technically starts with the former. If the paths are `org/foo/` and `org/foobar/`, you avoid that problem.